src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random(System.currentTimeMillis());
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int randomMaxValue = 100;
+    private final int defaultRadius = 10;
+
+    public final Figure getDefaultFigure() {
+        Color defaultColor = Color.WHITE;

```

Comment:
Create default color as constant, move it to class field:
private static final Color DEFAULT_COLOR = Color.WHITE;